### DISCUSSION TITLE
aws connector for github

### DISCUSSION THREAD

**Discussion by pjazdzyk** [community/community#36557] [Other Feature Feedback, Questions, & Ideas]
Hi,

I have tried today to create project on AWS using codestar. My intent was to link GitHub account with AWS as source repository. Everything goes fine, until a define connection tab appears. When I click "InstallNewApp" button I get Github unicorn page "his page is taking too long to load. Sorry about that. Please try refreshing and contact us if the problem persists."
I have checked in my github  application, and AWS connector for github is present and authorized. Did any on you have encountered such issue? I will be grateful for any suggestions or workaround. Thank you!

kind regards
P.

**Comment by ossanna16** [comment]
Hey there @pjazdzyk and thank you for asking a good question! I'm glad you found our community 🙂

**Reply by pjazdzyk** [reply]
What a great welcome I have got, thank you!

**Reply by ossanna16** [reply]
You are very welcome 🙂
